frontend/admin/src/components/Tenant/TenantEdit.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { EditProps, Edit, SimpleForm, TextInput, required } from 'react-admin'
+
+const TenantEdit: (props: EditProps) => JSX.Element = (props) => (
+  <Edit {...props}>
+    <SimpleForm>
+      <TextInput source="name" validate={required()} />
+      <TextInput source="email" validate={required()} />
```

Comment:
We currently don't have an email field for this in the models. Does it make sense from some specific use case perspective I am not thinking of to add it? If not please remove it.

Reply:
I noticed, that Tenant create has email and phone. I think we could remove them for now for the sake of simplicity.

Reply:
Removed fields, as requested.